experimenter/experimenter/nimbus-ui/src/components/PageResults/index.tsx

```diff
@@ -135,6 +139,9 @@ const PageResults: React.FunctionComponent<RouteComponentProps> = () => {
 
   const { external_config: externalConfig } = analysis.metadata || {};
 
+  const referenceBranchHelpMarkdown =
+    "Select the **reference branch** to set it as the reference for results. By default this is the experiment's configured reference branch.";
```

Comment:
Using "reference" here to mean both "the reference (branch)" and as "a reference for comparisons" is kind of awkward phrasing.

Reply:
Yea, fair. I'll see if I can come up with better phrasing here.